gcp_variant_transforms/libs/bigquery_vcf_schema_converter.py

```diff
@@ -222,24 +215,90 @@ def _add_format_fields(schema, formats):
   for field in schema.fields:
     if field.name in _CONSTANT_CALL_FIELDS:
       continue
-    formats.update({
-        field.name: _Format(field.name,
-                            parser.field_counts[vcfio.MISSING_FIELD_VALUE],
-                            _BIG_QUERY_TYPE_TO_VCF_TYPE_MAP.get(field.type),
-                            field.description)})
+    elif field.name in vcf_reserved_fields.FORMAT_FIELDS.keys():
+      reserved_definition = vcf_reserved_fields.FORMAT_FIELDS.get(field.name)
+      _validate_reserved_field(field, reserved_definition)
+      formats.update({field.name: reserved_definition})
+    else:
+      formats.update({field.name: _Format(
+          field.name,
+          bigquery_util.get_vcf_num_from_bigquery_schema(field.mode,
+                                                         field.type),
+          bigquery_util.get_vcf_type_from_bigquery_type(field.type),
+          field.description)})
+
+
+def _add_info_fields(field, infos):
+  # type: (bigquery.TableFieldSchema, Dict[str, _Info]) -> None
+  if field.name == bigquery_util.ColumnKeyConstants.ALTERNATE_BASES:
+    _add_info_fields_from_alternate_bases(field, infos)
+  elif field.name in vcf_reserved_fields.INFO_FIELDS.keys():
+    reserved_definition = vcf_reserved_fields.INFO_FIELDS.get(field.name)
+    _validate_reserved_field(field, reserved_definition)
+    infos.update({field.name: reserved_definition})
```

Comment:
I think it's better to use the description from the BQ schema, so something like:
```
infos.update({field.name: _Info(
    id=field.name,
    num=reserved_definition.num,
    type=reserved_definition.type,
    desc=field.description or reserved_definition.desc,
    source=None,
    version=None)})
```

Ditto for others. Also, please add `id=` etc so that it's more clear what the `None`s are for.

Reply:
Done. Thanks! Since we are not using the reserved_definition directly, I don't think it is necessary to use `Info` or `Format` for those reserved fields. Instead, I defined another namedtuple for it.